androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivity.java

```diff
@@ -113,6 +118,8 @@
     @Nullable
     private TwaLauncher mTwaLauncher;
 
+    private PackageManager mPackageManager;
```

Comment:
This doesn't need to be a member. (You can still make it a local variable in `onCreate if you like.)